module/mempool/stdmap/approvals.go

```diff
@@ -129,7 +183,6 @@ func (a *Approvals) Rem(resultID flow.Identifier, chunkIndex uint64) bool {
 	})
```

Comment:
The method you are handing to `Backend.Run` can error:
https://github.com/onflow/flow-go/blob/f6bb825fd5c1fbab48df2a10d09c31838cae05d7/module/mempool/stdmap/approvals.go#L173

This error would be lost, as you are not capturing the return value:
https://github.com/onflow/flow-go/blob/f6bb825fd5c1fbab48df2a10d09c31838cae05d7/module/mempool/stdmap/approvals.go#L165



You have a few options to fix this:
1.  instead doing a checked type cast 
    ```golang
     approvalMapEntity, ok := entity.(model.ApprovalMapEntity)
    ```
     you could do an unchecked type cast, which would panic in case it goes wrong. 
2. you capture the error return value an panic in line 183 if it is not nil